matplotcheck/base.py

```diff
@@ -105,13 +112,14 @@ def assert_plot_type(self, plot_type=None):
     """ TITLES TESTS/HELPER FUNCTIONS """
 
     def get_titles(self):
-        """Returns the suptitle (Figure title) and axes title of ax
+        """Returns the suptitle (Figure title) and axes title of `ax`.
```

Comment:
@ryla5068 when you built the docs locally, did `ax` highlight with only one set of ticks? rst is a bit different from markdown which is why i'm curious if that worked.

Reply:
It didn't highlight it, it just italicized it. I did this for every variable name I found in a description.

Reply:
perfect ! thanks! i'll resolve this comment.